README.md

```diff
@@ -8,7 +8,21 @@ Nginx example configuration can be found under [nginx](nginx).
 
 ## Demo
 
-**update the environment variables.** [.env.temp](.env.temp)
+**First update the environment variables.** [.env.temp](.env.temp)
+
+### Initialization
+
+### 1. Fast way of starting demo instance:
```

Comment:
please remove this from README. you can keep the script. but remove it from README - lets have just the **step by step**